redesign/generators.py

```diff
@@ -1,44 +1,55 @@
 """Contains generators for tabular, graph, and unstructured data profiles."""
 
+from synthetic_data.synthetic_data import make_data_from_report
+
 from .base_generator import BaseGenerator
 
```

Comment:
below we use relative, but here we use absolute. Ideally, we should be consistent with the approach. Not super big deal, but just calling it out.